File: 2º Periodo/AED II/Lab/lab01/iterativo.c
#include <stdio.h>
#include <string.h>
int main(){
	char s[100];
	char tag[]={"FIM\n"};
	int count;
	fgets(s, 100, stdin);
	while (strcmp(s,tag)!=0){
		count=0;
		for(int i=0;s[i]!='\0';i++){
			if(s[i]>='A' && s[i]<='Z') count++;
		}
		printf("%d\n",count);
		fgets(s, 100, stdin);
	}
	return 0;
}

File: 2º Periodo/AED II/Lab/lab01/iterativo.java
import java.util.Scanner;
public class iterativo{
	public static void main(String[] args){
		Scanner scanner = new Scanner(System.in);
		String tag="FIM";
		String s;
		s = scanner.nextLine();
		while(!s.equals(tag)){
			int count=0;
			for(int i=0;i<s.length();i++){
				char l = s.charAt(i);
				if(l>='A'&& l<='Z') count++;
			}
			System.out.println(count);
			s=scanner.nextLine();
		}
		scanner.close();
	}
}
